test/TestServerProjects/media_types/Generated/MediaTypesClient.cs

```diff
@@ -299,16 +299,17 @@ public virtual Response<string> BinaryBodyWithThreeContentTypes(ContentType1 con
             }
         }
 
-        /// <summary> Binary body with three content types. Pass in string &apos;hello, world&apos; with content type &apos;text/plain&apos;, {&apos;hello&apos;: world&apos;} with content type &apos;application/json&apos; and a byte string for &apos;application/octet-stream&apos;. </summary>
+        /// <summary> Body that&apos;s either text/plain or application/json. </summary>
+        /// <param name="contentType"> Upload file type. </param>
         /// <param name="message"> The payload body. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public virtual async Task<Response<string>> BinaryBodyWithThreeContentTypesAsync(string message, CancellationToken cancellationToken = default)
```

Comment:
why is the name changed here?

Reply:
This is not a name change. It is actually remove the BinaryBodyWithThreeContentTypes and add PutTextAndJsonBodyAsync. You could see the details here: https://github.com/Azure/autorest.csharp/pull/2072/files#r845862517